src/main/java/fr/xephi/authme/settings/properties/ProtectionSettings.java

```diff
@@ -24,14 +24,14 @@ public final class ProtectionSettings implements SettingsHolder {
     public static final Property<Boolean> ENABLE_GEOIP =
         newProperty("Protection.geoIpDatabase.enabled", true);
 
-    @Comment({"The MaxMind clientId used to download the GeoIp database,",
-        "get one at https://www.maxmind.com/en/accounts/current/license-key",
-        "The EssentialsX project has a very useful tutorial on how to generate",
-        "the license key: https://github.com/EssentialsX/Wiki/blob/master/GeoIP.md"})
+    @Comment({"Get the information needed for the GeoIP database at https://www.maxmind.com/en/accounts/current/license-key",
+        "This EssentialsX project has a very useful tutorial on how to generate it:",
```

Comment:
"This EssentialsX project" sounds weird, I would leave it as "The EssentialsX project" (there is only one)

Rest looks nice, thanks for the improvement!